mdtools.py

```diff
@@ -48,46 +58,57 @@ def mdrun(
     check_point_file: str = None,
     mdrun_options: list = None,
     gpus_per_node: int = None,
+    plumed_file: str = None,
+    use_api: bool = True
 ):
-    mpi_rank = max(mpi_rank - 1, 0)
+    mpi_rank = mpi_rank - 1
```

Comment:
I added this to make it work with one mpirank.
```suggestion
    mpi_rank = max(mpi_rank - 1,0)
```